session_items.py

```diff
@@ -1,73 +1,111 @@
 from flask import session
```

Comment:
We don't use this anymore (which means the name of this file is obsolete). Probably worth renaming to `trello_api.py` or something similar.